### PR TITLE
Schema: Notice event tags

### PR DESCRIPTION
**Pull request by medikoo**

### CODE REVIEW

**Review by Danwakeem — DISMISSED** [review]
LGTM 😎

**Review by Mmarzex — CHANGES_REQUESTED** [review]
I think this looks good @medikoo , but let's have @skierkowski take a look too since the naming we pick here will affect UX potentially.

**Review by Danwakeem — APPROVED** [review]
Looks good to me 👍

File: proto/serverless/instrumentation/tags/v1/notice.proto [hunk 1]
```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package serverless.instrumentation.tags.v1;
+
+option go_package = ".;protoc";
+
+message NoticeTags {
+    // Notice Message
+    string message = 1;
```
Comment (Mmarzex) [inline proto/serverless/instrumentation/tags/v1/notice.proto:9]: If we are going to have a single type here, can we get it written out what possible messages will be here given the shortened set of NoticeType's?

**Review by Mmarzex — CHANGES_REQUESTED** [review]

File: proto/serverless/instrumentation/tags/v1/notice.proto (hunk 1 above)
Comment (medikoo) [reply proto/serverless/instrumentation/tags/v1/notice.proto:9]: @max Do you mean to list here all the messages we configure in code (like `Binary body was removed`) and ensure this list is updated whenever we update the grammar of these messages or add new ones?

Comment (medikoo) [reply proto/serverless/instrumentation/tags/v1/notice.proto:9]: Ok, I think you rather mean listing fingerprints (message codes), as we should never rely on messages programmatically (they're either not used, or human readable messages for users)

**Review by Mmarzex — APPROVED** [review]

**Pull request merged**